.changeset/blue-plants-act.md

```diff
@@ -0,0 +1,14 @@
+---
+"@fluidframework/tree": minor
+"fluid-framework": minor
+---
+---
+"section": tree
+---
+
+Faster creation of large transactions and faster processing of inbound changes.
```

Comment:
We avoid punctuation in headings
```suggestion
Faster creation of large transactions and faster processing of inbound changes
```

Reply:
Ah, sorry, I would have made the change if it weren't for the auto-merge. I'll abstain from using it in the future.

Reply:
I have a fix up PR: https://github.com/microsoft/FluidFramework/pull/23895

Reply:
> Ah, sorry, I would have made the change if it weren't for the auto-merge. I'll abstain from using it in the future.

No worries 👍 auto merge is fine